CONTRIBUTING.md

```diff
@@ -1,13 +1,26 @@
 # How to Contribute
 
-One of the best ways to contribute is to participate in discussions and discuss issues. You can also contribute by submitting pull requests with code changes.
```

Comment:
Suggest leaving in the original sentence, but following it up with the addition, and then info for how to people can contribute without write access as Aaron commented.